src/helpers/index.ts

```diff
@@ -1 +1 @@
-export * from './ActivityAnalyzerTestHelper';
+// export * from './ActivityAnalyzerTestHelper';
```

Comment:
If you do not export the helpers, how can you retrieve it outside ? 
As importing using the full is not the best idea IMO (cf. you change higher: 
`import * as helpers from "@mediarithmics/plugins-nodejs-sdk/lib/helpers/ActivityAnalyzerTestHelper";`
@PoulpiFr : any opinion ?